BuildPropertiesPlugin/plugin/src/main/groovy/com/novoda/buildproperties/Entry.groovy

```diff
@@ -34,4 +34,29 @@ class Entry {
         value.call()
     }
 
+    Entry or(def fallback) {
+        def other = from(fallback)
+        new Entry(key, {
+            try {
+                return getValue()
+            } catch (Throwable e) {
+                try {
+                    return other.call()
+                } catch (Throwable e2) {
+                    throw CompositeException.from(e).add(e2)
```

Comment:
yes it is, but then you also need to clean up the stack trace of the wrapper exception to avoid showing misleading traces like 
```
at com.novoda.buildproperties.CompositeException.add(CompositeException.groovy:30)
	at com.novoda.buildproperties.CompositeException$add$0.call(Unknown Source)
	at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCall(CallSiteArray.java:48)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:113)
	at org.codehaus.groovy.runtime.callsite.AbstractCallSite.call(AbstractCallSite.java:125)
```
As I said is a matter of amount effort needed vs actual gain.